### ISSUE TITLE
Could not pass event PlayerJoinEvent #274 Build

### ISSUE THREAD

**Issue by dimacraftsv** [DSH105/HoloAPI#54]
Build #274
Server Version: 1.7.2

http://pastebin.com/E1vGN3WN

Thanks

**Comment by stuntguy3000** [comment]
Is there anything stopping you from updating?

**Comment by dimacraftsv** [comment]
I dont get it, can you explain?

edit: got it now, this is only compatible with 1.7.8, so this is not a bug, I will close this issue

I will not update to 1.7.8 yet, 90% of 54 plugins that I use are not compatible with this versions, that's why...

**Comment by stuntguy3000** [comment]
"edit: got it now, this is only compatible with 1.7.8, so this is not a bug, I will close this issue

I will not update to 1.7.8 yet, **90% of 54** plugins that I use are not compatible with this versions, that's why..."

Almost every plugin is 1.7.9 compatible
.

**Comment by dimacraftsv** [comment]
Almost it's not all, it's not that simple to upgrade a server that was already running for 3~4 months with an older version.

I'll give just one example: the new GriefPrevention is essential on my server, but the newer version that is compatible with 1.7.9 is unstable and I'm helping reporting the issues that I had on my tests. This is only one of the many others that are NOT COMPATIBLE with 1.7.9!

So I will not update yet, a single plugin is enought to crush the entire gameplay.

**Comment by DSH105** [comment]
@stuntguy3000 Not every plugin for 1.7.2 is updated to 1.7.8/9 yet.

@dimacraftsv Has your issue been resolved yet, seeing as you closed this ticket? :)

**Comment by dimacraftsv** [comment]
@DSH105  nope, I'm using an older build (1.20), newer versions does not work on 1.7.2

I thought it was meant to be this way, I'll reopen this Issue and wait for your response, sorry!

**Comment by DSH105** [comment]
@dimacraftsv Are the later builds working yet?
